.github/workflows/gradle-pr-build.yml

```diff
@@ -26,4 +26,4 @@ jobs:
           path: 'ktlint/build/reports/tests/**'
           if-no-files-found: warn
       - name: Build with dev Kotlin version
-        run: ./gradlew -PkotlinDev clean build ktlint
+        run: ./gradlew -PkotlinDev clean build ktlint --no-daemon
```

Comment:
https://github.com/pinterest/ktlint/runs/7863303673?check_suite_focus=true

```
FAILURE: Build failed with an exception.

* What went wrong:
Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)
```

Reply:
https://github.com/pinterest/ktlint/runs/7883648212?check_suite_focus=true